source/Calamari.Common/Features/Packages/NuGet/NupkgExtractor.cs

```diff
@@ -44,12 +44,15 @@ public int Extract(string packageFile, string directory)
             {
                 foreach (var entry in archive.Entries)
                 {
+                    if (entry.Key == null)
+                        continue;
+                    
                     var unescapedKey = UnescapePath(entry.Key);
 
                     if (IsExcludedPath(unescapedKey))
                         continue;
 
-                    var targetDirectory = Path.Combine(directory, Path.GetDirectoryName(unescapedKey));
+                    var targetDirectory = Path.Combine(directory, Path.GetDirectoryName(unescapedKey) ?? string.Empty);
```

Comment:
if `Path.Combine` gets an empty string, it just returns the other parameter (`directory` in this case)